src/components/_LuxLibraryContactInfo.vue

```diff
@@ -90,7 +88,6 @@ export default {
     background-color: var(--color-white);
     a {
       color: var(--color-rich-black);
-      &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}